src/transactions/SponsorshipUtils.cpp

```diff
@@ -183,7 +183,7 @@ isSignerSponsored(std::vector<Signer>::const_iterator const& signerIt,
 // Utility functions to check if you can establish/remove/transfer sponsorships
 //
 ////////////////////////////////////////////////////////////////////////////////
-static uint32_t
+uint32_t
```

Comment:
note that we also 2 other implementations of this function --- and I just opened https://github.com/stellar/stellar-core/pull/3192 because returning uint32 is a footgun. So you probably want to move to int32 and use that util in test code as well after that other PR gets merged?

Reply:
Done. I moved the test implementation into `TestUtils` to keep it separate. After all, these tests are testing `computeMultiplier` as well.